src/main/java/org/broadinstitute/consent/http/db/UserPropertyDAO.java

```diff
@@ -44,14 +44,13 @@
     void deletePropertyByUser(@BindList("propertyKeyList") List<String> propertyKeyList, @Bind("userId") Integer userId);
 
     @SqlQuery(value = "SELECT * FROM user_property WHERE " +
-            "(propertykey = '" + INSTITUTION + "' AND propertyvalue != :institutionName) OR " +
             "(propertykey = '" + ARE_YOU_PRINCIPAL_INVESTIGATOR + "' AND  propertyvalue != :isThePI) OR " +
             "(propertykey = '" + DO_YOU_HAVE_PI + "' AND  propertyvalue != :havePI) OR " +
             "(propertykey = '" + ERA_COMMONS_ID + "' AND  propertyvalue != :eRACommonsID) OR " +
             "(propertykey = '" + PUBMED_ID + "' AND  propertyvalue != :pubmedID) OR " +
             "(propertykey = '" + SCIENTIFIC_URL + "' AND  propertyvalue != :scientificURL) " +
             " AND userid = :userId")
-    List<UserProperty> findResearcherProperties(@Bind("userId") Integer userId, @Bind("institutionName") String institutionName,@Bind("isThePI") String isThePI,
```

Comment:
only used in researcherservice.hasupdatedfields , and field is not needed for that method